lib/linter/interpolate.js

```diff
@@ -9,20 +9,27 @@
 // Public Interface
 //------------------------------------------------------------------------------
 
-module.exports = (text, data) => {
-    if (!data) {
-        return text;
-    }
+module.exports = {
+    getPlaceholderMatcher() {
+        return /\{\{([^{}]+?)\}\}/gu;
+    },
+    interpolate(text, data) {
+        if (!data) {
+            return text;
+        }
 
-    // Substitution content for any {{ }} markers.
-    return text.replace(/\{\{([^{}]+?)\}\}/gu, (fullMatch, termWithWhitespace) => {
-        const term = termWithWhitespace.trim();
+        const matcher = module.exports.getPlaceholderMatcher();
```

Comment:
To avoid confusion.
```suggestion
        const matcher = this.getPlaceholderMatcher();
```

Reply:
`this` wouldn't work unless the consuming modules always import the whole object and call this function as a method through that object.

I think it would be best to simply declare and export two functions:

```js
function getPlaceholderMatcher() {
   // ...
}

function interpolate() {
    // .... just call getPlaceholderMatcher() as a function here.
}

module.exports = {
    getPlaceholderMatcher,
    interpolate
}
```